airbyte-local.sh

```diff
@@ -141,12 +132,12 @@ function writeSrcCatalog() {
                --argjson src_catalog_overrides "$src_catalog_overrides" '{
               streams: [
                 .catalog.streams[]
-                  | select($src_catalog_overrides[.name] != "disabled")
-                  | .incremental = ((.supported_sync_modes|contains(["incremental"])) and ($src_catalog_overrides[.name] != "full_refresh") and ($full_refresh != "true"))
+                  | select($src_catalog_overrides[.name].disabled != true)
+                  | .incremental = ((.supported_sync_modes|contains(["incremental"])) and ($src_catalog_overrides[.name].sync_mode != "full_refresh") and ($full_refresh != "true"))
                   | {
                       stream: {name: .name},
                       sync_mode: (if .incremental then "incremental" else "full_refresh" end),
-                      destination_sync_mode: (if .incremental then "append" else "overwrite" end)
+                      destination_sync_mode: ($src_catalog_overrides[.name].destination_sync_mode? // if .incremental then "append" else "overwrite" end)
```

Comment:
What does `//` do here?

Reply:
If left side of `//` is null, it executes the right side.